src/globals/mixins.scss

```diff
@@ -544,8 +544,8 @@
   background-color: $colour-white;
   border-color: $colour-white;
   &:hover {
-    color: $colour-white;
-    background-color: $colour-primary;
+    color: $colour-black;
+    background-color: $colour-brand-3;
```

Comment:
$colour-brand-3 is green #50B400

I won't do it now, but it's making me think maybe we should rename the variable to `$colour-brand-3`. We call it green when we're talking about the design, we don't call it "Brand Colour #3". Calling it green in the code two would just make the code a little bit easier to read and write.

Reply:
https://www.color-hex.com/color/50b400